components/SwipeRow.js

```diff
@@ -134,24 +140,35 @@ class SwipeRow extends Component {
 		}
 	}
 
-	handlePanResponderEnd(e, gestureState) {
-		// re-enable scrolling on listView parent
+	ensureScrollEnabled() {
 		if (!this.parentScrollEnabled) {
 			this.parentScrollEnabled = true;
 			this.props.setScrollEnabled && this.props.setScrollEnabled(true);
 		}
+	}
+
+	handlePanResponderEnd(e, gestureState) {
+
+		// decide how much the velocity will affect the final position that the list item settles in.
+		const swipeToOpenVelocityContribution = this.props.swipeToOpenVelocityContribution;
+		const possibleExtraPixels = this.props.rightOpenValue * (swipeToOpenVelocityContribution);
+		const clampedVelocity = gestureState.vx > 5 ? 5 : gestureState.vx
+		const projectedExtraPixels = possibleExtraPixels * (clampedVelocity / 5);
+
+		// re-enable scrolling on listView parent
+		this._ensureScrollEnabledTimer = setTimeout(this.ensureScrollEnabled, 300);
```

Comment:
Can we move this one to the top as a `const` as well. Maybe:

```
const SCOLL_LOCK_MILLISECONDS = 300;
```

Reply:
Yeah, I might try to spell it right this time too!

Reply:
oh i see, these are new consts to be introduced.